app/assets/stylesheets/application.css.scss

```diff
@@ -63,6 +63,23 @@ footer {
   padding: 10px;
 }
 
+.splash-title {
+  font-family: 'Miltonian Tattoo', cursive;
+  text-align: center;
+  margin: 0px;
+}
+
+.splash-body {
+  font-family: 'Miltonian Tattoo', cursive;
+  text-align: center;
+  a {
+    color: black;
+  }
+  a:hover {
+    color: darken(blue, 30%)
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Declaration should be terminated by a semicolon